input-output-hk/cardano-sl
# [CSM-426] Serve Swagger doc from `wallet` endpoint

How to test?

1. Build everything by running `./scripts/build/cardano-sl.sh`
2. Run wallet api
```
tmux
scripts/launch/demo-with-wallet-api.sh
```
3. Test endpoint w/ `curl`
```
curl --cacert {xxx}/cardano-sl/scripts/tls-files/ca.crt -v https://localhost:8090/docs
```
Or even better: Open https://localhost:8090/docs in your browser (ignore unsafe certificate warning).

You should see something like this:

![swagger-wallet](https://user-images.githubusercontent.com/47693/29829526-f5b80344-8cdf-11e7-8479-fa18cf16f69a.png)
